card/spec/models/card_spec.rb

```diff
@@ -166,7 +166,7 @@ def assert_stable card1
 
 
   it 'create_without_read_permission' do
-    c = Card.create!({:name=>"Banana", :type=>"Fruit", :content=>"mush"})
+    c = Card.create!({name: "Banana", type: "Fruit", content: "mush"})
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.